packages/blade/src/components/Input/SelectInput/SelectInput.tsx

```diff
@@ -107,7 +107,9 @@ const _SelectInput = (
         isPopupExpanded={isOpen}
         onClick={onTriggerClick}
         onKeyDown={onTriggerKeydown}
-        onBlur={onTriggerBlur}
+        onBlur={({ name }) => {
+          onTriggerBlur?.({ name, value, userOnBlur: onBlur });
```

Comment:
nit: `callback` seems more appropriate than `userOnBlur`. 
```suggestion
          onTriggerBlur?.({ name, value, onBlurCallback: onBlur });
```